src/Plugin/Patches.php

```diff
@@ -76,6 +76,11 @@ class Patches implements PluginInterface, EventSubscriberInterface, Capable
      */
     protected $patchCollection;
 
+    /**
+     * @var array
+     */
+    private $installedPatches;
```

Comment:
Was this mistakenly added?

Reply:
Nope, `$this->installedPatches` is used on line 97 of the same file :-)